### ISSUE TITLE
systemctl command in logrotate file on el6

### ISSUE THREAD

**Issue by cgtx** [iuscommunity-pkg/httpd24u#2]
The logrotate file contains the systemctl command, which isn't valid on el6.

```
/var/log/httpd/*log {
    missingok
    notifempty
    sharedscripts
    delaycompress
    postrotate
        /bin/systemctl reload httpd.service > /dev/null 2>/dev/null || true
    endscript
}
```

**Comment by cgtx** [comment]
Fix released in build `httpd24u-2.4.12-4.ius`, which will move to the testing repos tonight.

**Comment by cgtx** [comment]
This package will move to the stable repos tonight.
